src/core/Akka.Persistence.TCK/Snapshot/SnapshotStoreSpec.cs

```diff
@@ -184,7 +185,7 @@ public virtual void SnapshotStore_should_load_the_most_recent_snapshot_matching_
         public virtual void SnapshotStore_should_delete_a_single_snapshot_identified_by_SequenceNr_in_snapshot_metadata()
         {
             var md = Metadata[2];
-            md = new SnapshotMetadata(md.PersistenceId, md.SequenceNr, Sys.Scheduler.Now.DateTime);
+            md = new SnapshotMetadata(md.PersistenceId, md.SequenceNr,  md.Timestamp);
```

Comment:
Basic issue with the test before was that we were using the wrong timestamp so `Metadata[2];` never got deleted because now timestamps matter. Adjusted the `SnapshotMetadata` to use the same timestamp as the targeted snapshot so the test assertions pass without being changed.